src/lib/http/client/client.ts

```diff
@@ -26,7 +26,7 @@ export class FetchClient implements Client {
 	}
 
 	private resolveUrl(baseUrl: URL, endpoint: string, query?: QueryParameters): string {
-		let url = `${baseUrl}/${endpoint}`.replaceAll(/(?<!:)\/{2,}/g, '/');
+		let url = `${baseUrl}/${endpoint}`.replaceAll(/([^:]\/)\/+/g, "$1");
```

Comment:
why "$1" here?

Reply:
`$1` is the first group from your regular expression

Reply:
That makes sense